deploy/helm/kubecf/assets/operations/instance_groups/uaa.yaml

```diff
@@ -60,11 +60,11 @@
               name: {{ printf "%s.var-router-ssl" .Release.Name | quote }}
               key: ca
         - name: CF_USERNAME
-          value: admin
+          value: uaa_setup
```

Comment:
Do we need a separate account for this? Is `credhub_setup` no good for this?  I realize that this is in the UAA group, but it's only used if credhub is enabled (and is only used to help credhub).

Reply:
Even though the outcome of this setup is to help credhub, there's no credhub specifics here other than it being on a conditional block only if credhub is enabled. I'd prefer to keeping it this way.

Reply:
Yeah, I was thinking about not needing more credentials around that could get leaked, but I guess not _really_ since they should be similar in difficulty to exfiltrate.